Fix GuardOnDataDependentSymNode in the normalize operator

henryoier opened the pull request:
Test Plan:
Dumped the local net torch.package to local

Ran
```
buck2 run scripts/shengqin:test_model_export -- /tmp/mtia_local_torch_package {\"local\":null}
```
succeeded

Reviewed By: hongyang-zhao

Differential Revision: D73405271

henryoier commented:
@pytorchbot label "topic: not user facing"

houseroad commented on torch/_refs/linalg/__init__.py:
```diff
@@ -149,7 +151,7 @@ def vector_norm(
 
     to_result_dtype = partial(_maybe_convert_to_dtype, dtype=result_dtype)
-
+ 
```
remove this empty space?

Also the lint error is related.

henryoier replied:
Manually resolved it. Let's see how the github check goes

houseroad reviewed (approved):
Please fix the lint error before landing.

henryoier committed:
Fix GuardOnDataDependentSymNode in the normalize operator (#152039)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/152039

Test Plan:
Dumped the local net torch.package to local

Ran
```
buck2 run scripts/shengqin:test_model_export -- /tmp/mtia_local_torch_package {\"local\":null}
```
succeeded

Reviewed By: hongyang-zhao, houseroad

Differential Revision: D73405271
